src/main/scala/swiss/dasch/api/ProjectsEndpointsHandler.scala

```diff
@@ -122,10 +123,12 @@ final case class ProjectsEndpointsHandler(
             prj <- projectService.findOrCreateProject(shortcode).mapError(InternalServerError(_))
             tmpDir <-
               storageService.createTempDirectoryScoped(s"${prj.shortcode}-ingest").mapError(InternalServerError(_))
-            tmpFile = tmpDir / filename.value
-            _      <- stream.run(ZSink.fromFile(tmpFile.toFile)).mapError(InternalServerError(_))
-            asset  <- ingestService.ingestFile(tmpFile, shortcode).mapError(InternalServerError(_))
-            info   <- assetInfoService.findByAssetRef(asset.ref).someOrFailException.orDie
+            tmpFile   = tmpDir / filename.value
+            _        <- stream.run(ZSink.fromFile(tmpFile.toFile)).mapError(InternalServerError(_))
+            fileSize <- Files.size(tmpFile).map(_ == 0).mapError(InternalServerError(_))
+            _        <- ZIO.fail(InternalServerError("file size zero")).when(fileSize)
```

Comment:
nitpick: arguably, this variably should be named `isEmpty` or so, because it does not really hold the file size. But it should be clear enough as is.

Reply:
Changed.